plugin/src/main/java/net/dzikoysk/funnyguilds/config/PluginConfiguration.java

```diff
@@ -1144,6 +1144,14 @@ public static class LivesRepeatingSymbol extends OkaeriConfig {
     @Comment("Jak długo plugin powinien czekać na zatrzymanie wszystkich bieżących zadań przy wyłączaniu serwera")
     public Duration pluginTaskTerminationTimeout = Duration.ofSeconds(30);
 
+    @Comment("")
+    @Comment("Jaki argument powinien zostać podany przez gracza gdy chce zaprosić wszystkich graczy w danym promieniu")
+    public String inviteCommandAllArgument = "*";
```

Comment:
Niby fajnie, że można zmienić. Moim zdaniem jednak takie rzeczy powodują, że config powoli przestaje być managable i nikt nie ma pojęcia, co tutaj się dzieje — zarówno developerzy jak i użytkownicy, a administrator nawet jak poradzi sobie, to musi się przekopywać przez pełno zbędnych opcji.

Reply:
Kamilkime to ładnie opisał - `@Comment("Maksymalna odległość, w jakiej zapraszani są gracze w momencie użycia komendy \"/invite *\"")`